src/modules/apps/install.js

```diff
@@ -8,33 +8,44 @@ import {
   wildVersionPattern,
 } from '../../manifest'
 
+const ARGS_START_INDEX = 2
+
+function installApps (apps) {
+  const app = apps.shift() || `${manifest.vendor}.${manifest.name}@${manifest.version}`
+  log.debug('Starting to install app', app)
+  const appRegex = new RegExp(`^${vendorPattern}.${namePattern}@${wildVersionPattern}$`)
+  if (!appRegex.test(app)) {
+    log.error('Invalid app format, please use <vendor>.<name>@<version>')
+    return Promise.resolve()
+  }
+
+  return installApp(app)
+  .then(() => log.info(`Installed app ${app} successfully`))
+  .then(() =>
+    apps.length > 0
+      ? installApps(apps)
+      : Promise.resolve()
+  )
+  .catch(err => {
+    if (err.statusCode === 409) {
+      return log.error(`App ${app} already installed`)
+    }
+    return Promise.reject(err)
```

Comment:
Let's say I ask for `vtex install foo bar zaz`, then `foo` succeeds, but `bar` fails. This way, `zaz` would be ignored. I think that either `zaz` should be installed anyway, or at least the toolbelt should warn that it was not installed due to previous errors.

Something like:
```sh
$ vtex install foo bar zas
info:    foo installed
error:   could not install bar because ...
warn:    skipped installation of zaz due to previous errors
```